composer.json

```diff
@@ -30,7 +30,7 @@
     "require-dev": {
         "mockery/mockery": "^1.3",
         "orchestra/testbench": "^5.0 || ^6.0",
-        "phpunit/phpunit": "^8.5"
+        "phpunit/phpunit": "^8.5 || ^9.0"
```

Comment:
These aren't actually installed in end users apps, but good to allow it in dev anyway :)